app/javascript/packages/address-search/components/in-person-locations.tsx

```diff
@@ -37,7 +39,8 @@ function InPersonLocations({ locations, onSelect, address }: InPersonLocationsPr
             count: locations?.length,
           })}
       </h3>
-      <p>{t('in_person_proofing.body.location.po_search.results_instructions')}</p>
+      {infoAlertURL && <InPersonLocationRedirectAlert infoAlertURL={infoAlertURL} />}
```

Comment:
[I had originally implemented this](https://github.com/18F/identity-idp/pull/9172/commits/c9e77a513d603fc2dd8ce55477346702d9bb3acd) as a component that didn't take any parameters; a reference was passed to the `InPersonLocations` component, which rendered it if it was passed.

Then I realized that the alert couldn't be self-contained; it needs to have a link to a page on the brochure site, and the PO search component as rendered on the brochure site wouldn't have access to the `MarketingSiteContext` helper to construct the link.

So, since the component needed a URL to render, I decided to make the URL the toggle for whether to display the alert or not. We follow this same pattern for deciding whether to render the Select buttons depending on the presence of the `onSelect` parameter.

Reply:
changed the approach in 331753c1672554f80d929798cd2fb8a81ad45ca9